crates/glaredb/src/local.rs

```diff
@@ -139,11 +144,15 @@ impl LocalSession {
         )
         .await?;
 
-        // TODO: Make this configurable through client commands.
-        let sess = if let Ok(url) = std::env::var("RPC_HOST_URL") {
```

Comment:
This effectively drops "support" for that env var, right?

Reply:
Yeah, that env var was always a temporary thing to test it out. I planned on removing it.

Reply:
Yea no prob, I never got around to using it just thought i'd double-check